### PR TITLE
VPN-5018 enable cryptosettings for android

### PR DESCRIPTION
**Pull request by strseb**
- Fix test code
- Add cryptosettings
- Pile of shame added!

## Description

    Describe your changes

## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt [hunk 1]
```diff
@@ -37,14 +38,14 @@ class VPNService : android.net.VpnService() {
     private var mBackgroundPingTimerMSec: Long = 3 * 60 * 60 * 1000 // 3 hours, in milliseconds
     private val mMetricsTimer: CountDownTimer = object : CountDownTimer(
         mBackgroundPingTimerMSec,
-        mBackgroundPingTimerMSec / 4
```
Comment (strseb) [inline android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt:40]: Huh sorry, ktlint added those things

File: android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNServiceBinder.kt [hunk 2]
```diff
@@ -12,6 +12,7 @@ import android.os.Parcel
 import kotlinx.serialization.decodeFromString
 import kotlinx.serialization.json.Json
 import org.json.JSONObject
+import org.mozilla.firefox.qt.common.Prefs
```
Comment (brizental) [inline android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNServiceBinder.kt:15]: Why add these import if this file has no changes?

File: android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/BootReceiver.kt [hunk 3]
```diff
@@ -8,6 +8,7 @@ import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
 import android.os.Build
+import org.mozilla.firefox.qt.common.Prefs
```
Comment (brizental) [inline android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/BootReceiver.kt:11]: Why add these import if this file has no changes?

**Review by brizental — APPROVED** [review]

File: android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/BootReceiver.kt (hunk 3 above)
Comment (strseb) [reply android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/BootReceiver.kt:11]: The `Prefs` moved from `/daemon`to `/common` therefore i need to import it :)

**Comment by cure53** [comment]
:heavy_check_mark: LGTM

**Pull request merged**